app/views/faculty_surveys/new.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="mx-auto w-full">
+  <h1>New faculty survey</h1>
```

Comment:
Please make the letter 'f' and 's' upper case in "faculty" and "survey" respectively. It is not that relevant but just to be consistent with the title you have in place for other pages.

Reply:
Thanks for noticing - done.